README.md

```diff
@@ -219,3 +234,59 @@ let datetime = &(DATE.to_owned() + T + TIME);
 let datetime = &(DATE.to_string() + T + TIME);
 ```
 
+## Additional macro benches
+
+### `concat_string_macro`
+
+* Rank: #1 @10ns
+* Crate: https://crates.io/crates/concat-string
+
+```rust
+#[macro_use(concat_string)]
+extern crate concat_string;
+let datetime = concat_string!(DATE, T, TIME);
+```
+
+### `concat_strs_macro`
+
+* Rank: #1 @10ns
+* Crate: https://crates.io/crates/concat_strs
+
+Unfortunately, this macro [breaks RustAnalyzer](https://github.com/rust-analyzer/rust-analyzer/issues/6835).
+
+```rust
+#[macro_use(concat_strs)]
+extern crate concat_strs;
+let datetime = &concat_strs!(DATE, T, TIME);
+```
+
+### `string_concat_macro`
+
+* Rank: #1 @10ns
+* Crate: https://crates.io/crates/string_concat
+
+```rust
+#[macro_use]
+extern crate string_concat;
+let datetime = &string_concat::string_concat!(DATE, T, TIME);
+```
+
+### `concat_in_place_macro`
+
+* Rank: #2 @14ns
+* Crate: https://crates.io/crates/concat-in-place
+
+```rust
+let datetime = concat_in_place::strcat!(&mut url, DATE T TIME);
+```
+
+### `joinery`
+
+* Rank: #22 @57ns
+* Crate: https://crates.io/crates/joinery
+
+```rust
+use joinery::prelude::*;
+let vec = vec![DATE, T, TIME];
+let datetime = &vec.iter().join_with("").to_string();
```

Comment:
Recommend instead, to avoid penalties:

Version 1:

```rust
use joinery::prelude::*;
let datetime = [DATE, T, TIME].join_concat().to_string();
```

Version 2:

```rust
use joinery::prelude::*;
let datetime = [DATE, TIME].join_with(T).to_string();
```